formats/placeholder/publisher/details.json

```diff
@@ -0,0 +1,39 @@
+{
+  "$schema": "http://json-schema.org/draft-04/schema#",
+  "type": "object",
+  "additionalProperties": false,
```

Comment:
Does this mean that placeholder items are restricted in what can be in the details hash?  I'm not sure that's necessarily what we want.

A placeholder item could readily have the full details that the eventual non-placeholder format will have, and only be using the placeholder variant to prevent routes being registered during the transition phase.